auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -82,9 +81,25 @@ public Boolean nullable() {
 
   @Override
   public boolean applicable(Context context) {
+    TypeMirror autoValueClass = context.autoValueClass().asType();
+    // Disallow manual implementation of the CREATOR instance
+    if (!needsCreator(context)) {
+      context.processingEnvironment().getMessager().printMessage(Diagnostic.Kind.ERROR,
+          "Manual implementation of a static Parcelable.Creator<T> CREATOR field found when processing "
+              + autoValueClass.toString() + ". Remove this so auto-value-parcel can automatically generate the "
+              + "implementation for you.");
+      return false;
```

Comment:
I don't think either of these branches need to early exit. The rest of the class can still be processed normally despite having the static field or the writeToParcel method.